setup.py

```diff
@@ -1,17 +1,29 @@
 # -*- coding: utf-8 -*-
+################################################################################
+# Copyright (c), AiiDA team and individual contributors.                       #
+#  All rights reserved.                                                        #
+# This file is part of the AiiDA-wannier90 code.                               #
+#                                                                              #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-wannier90 #
+# For further information on the license, see the LICENSE.txt file             #
+################################################################################
 
 from __future__ import absolute_import
 import json
+import os
 
 from setuptools import setup, find_packages
 
 if __name__ == '__main__':
+    THIS_FOLDER = os.path.split(os.path.abspath(__file__))[0]
     with open('setup.json', 'r') as info:
         kwargs = json.load(info)
     setup(
         include_package_data=True,
         setup_requires=['reentry'],
         reentry_register=True,
+        long_description=open(os.path.join(THIS_FOLDER, 'README.md')).read(),
```

Comment:
We should use `open` with `encoding='utf-8'`, in case there are non-ASCII characters in the README.

Unfortunately, this error will not be caught on most platforms - it depends on the default encoding.. speaking from experience 😄